src/Application/Email/Config.php

```diff
@@ -4,14 +4,12 @@
 
 namespace Mailer\Application\Email;
 
+use JetBrains\PhpStorm\Pure;
 use Mailer\Application\ConfigModels\Email;
 
 class Config
 {
-    /** @var Email[] Keyed on template key */
-    private array $emailSettings;
-
-    public function __construct(array $emailSettings)
+    public function __construct(private array $emailSettings)
```

Comment:
Keep doc comment, and more specific type hint is probably useful to keep in a doc block here?